src/lib/components/radio-button/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { useTheme } from '@emotion/react';
+import stylin from '@stylin.js/react';
+import React, { FC, PropsWithChildren, useState } from 'react';
+
+import { Box } from '../../elements';
+import { RadioCircleSVG } from '../../icons';
+import { Theme } from '../../theme';
+import {
+  RadioButtonElementProps,
+  RadioButtonProps,
+} from './radio-button.types';
+
+export const RadioButton: FC<PropsWithChildren<RadioButtonProps>> = ({
+  ...props
+}) => {
+  const { dark } = useTheme() as Theme;
```

Comment:
You don't really need to check the theme in this case

Reply:
check again Sir